web/src/test/java/it/pagopa/selfcare/dashboard/web/security/ExchangeTokenServiceTest.java

```diff
@@ -75,7 +79,7 @@ void exchange_illegalBase64Signature() {
         ExchangeTokenProperties properties = new ExchangeTokenProperties();
         properties.setSigningKey(jwtSigningKey);
         // when
-        Executable executable = () -> new ExchangeTokenService(null, null, null, null, properties);
+        Executable executable = () -> new ExchangeTokenService(null, null, null, null, properties, null);
```

Comment:
it might be better to inject mock as variable class avoiding replicate new ExchangeTokenService

Reply:
Some of the tests are specifically made to test the parsing of the signing key in the  istantiation of the ExchangeTokenService constructor